src/storage/txn/scheduler.rs

```diff
@@ -878,8 +880,12 @@ impl<E: Engine, L: LockManager> TxnScheduler<E, L> {
             SCHED_STAGE_COUNTER_VEC.get(tag).write_finish.inc();
         }
 
-        debug!("write command finished";
-            "cid" => cid, "pipelined" => pipelined, "async_apply_prewrite" => async_apply_prewrite);
+        debug_with_req_info_tracker!("write_command_finished",
+            [sched_details.tracker],
```

Comment:
How about implementing `slog::Value for [Tracker]`? Then we can unify log styles.

```rust
debug!("msg"; "trackers" => sched_details.tracker);
```

Reply:
Thanks for the advice, PTAL